tidb-cloud/import-csv-files.md

```diff
@@ -197,6 +199,58 @@ To import the CSV files to TiDB Cloud, take the following steps:
 
 </div>
 
+<div label="Azure Blob Storage">
+
+1. Open the **Import** page for your target cluster.
+
+    1. Log in to the [TiDB Cloud console](https://tidbcloud.com/) and navigate to the [**Clusters**](https://tidbcloud.com/console/clusters) page of your project.
+
+        > **Tip:**
+        >
+        > If you have multiple projects, you can click <MDSvgIcon name="icon-left-projects" /> in the lower-left corner and switch to another project.
+
+    2. Click the name of your target cluster to go to its overview page, and then click **Import** in the left navigation pane.
+
+2. Click **Import Data** in the upper-right corner.
+
+    If this is your first time importing data into this cluster, select **Import From Azure Blob Storage**.
+
+3. On the **Import Data from Azure Blob Storage** page, provide the following information for the source CSV files:
+
+    - **Import File Count**: select **One file** or **Multiple files** as needed.
```

Comment:
This option is not on the UI.
![image](https://github.com/user-attachments/assets/ad7fc52b-d739-4145-9382-255daac9e21c)